fiber.toml

```diff
@@ -65,9 +65,120 @@ peer_list_url = "https://downloads.skycoin.net/blockchain/peers.txt"
+    "2efrft5Lnwjtk7F1p9d7BnPd72zko2hQWNi",
+    "Syzmb3MiMoiNVpqFdQ38hWgffHg86D2J4e",
+    "2g3GUmTQooLrNHaRDhKtLU8rWLz36Beow7F",
+   	"D3phtGr9iv6238b3zYXq6VgwrzwvfRzWZQ",
+    "gpqsFSuMCZmsjPc6Rtgy1FmLx424tH86My",
+    "2EUF3GPEUmfocnUc1w6YPtqXVCy3UZA4rAq",
+    "TtAaxB3qGz5zEAhhiGkBY9VPV7cekhvRYS",
+   	"2fM5gVpi7XaiMPm4i29zddTNkmrKe6TzhVZ",
+   	"ix3NDKgxfYYANKAb5kbmwBYXPrkAsha7uG",
+    "2RkPshpFFrkuaP98GprLtgHFTGvPY5e6wCK",
+    "Ak1qCDNudRxZVvcW6YDAdD9jpYNNStAVqm",
+    "2eZYSbzBKJ7QCL4kd5LSqV478rJQGb4UNkf",
+    "KPfqM6S96WtRLMuSy4XLfVwymVqivdcDoM",
+    "5B98bU1nsedGJBdRD5wLtq7Z8t8ZXio8u5",
+    "2iZWk5tmBynWxj2PpAFyiZzEws9qSnG3a6n",
+    "XUGdPaVnMh7jtzPe3zkrf9FKh5nztFnQU5",
+    "hSNgHgewJme8uaHrEuKubHYtYSDckD6hpf",
+    "2DeK765jLgnMweYrMp1NaYHfzxumfR1PaQN",
+    "orrAssY5V2HuQAbW9K6WktFrGieq2m23pr",
+    "4Ebf4PkG9QEnQTm4MVvaZvJV6Y9av3jhgb",
+    "7Uf5xJ3GkiEKaLxC2WmJ1t6SeekJeBdJfu",
+   	"oz4ytDKbCqpgjW3LPc52pW2CaK2gxCcWmL",
+   	"2ex5Z7TufQ5Z8xv5mXe53fSQRfUr35SSo7Q",
+    "WV2ap7ZubTxeDdmEZ1Xo7ufGMkekLWikJu",
+    "ckCTV4r1pNuz6j2VBRHhaJN9HsCLY7muLV",
+    "MXJx96ZJVSjktgeYZpVK8vn1H3xWP8ooq5",
+    "wyQVmno9aBJZmQ99nDSLoYWwp7YDJCWsrH",
+   	"2cc9wKxCsFNRkoAQDAoHke3ZoyL1mSV14cj",
+   	"29k9g3F5AYfVaa1joE1PpZjBED6hQXes8Mm",
+    "2XPLzz4ZLf1A9ykyTCjW5gEmVjnWa8CuatH",
+    "iH7DqqojTgUn2JxmY9hgFp165Nk7wKfan9",
+    "RJzzwUs3c9C8Y7NFYzNfFoqiUKeBhBfPki",
+    "2W2cGyiCRM4nwmmiGPgMuGaPGeBzEm7VZPn",
+    "ALJVNKYL7WGxFBSriiZuwZKWD4b7fbV1od",
+    "tBaeg9zE2sgmw5ZQENaPPYd6jfwpVpGTzS",
+    "2hdTw5Hk3rsgpZjvk8TyKcCZoRVXU5QVrUt",
+    "A1QU6jKq8YgTP79M8fwZNHUZc7hConFKmy",
+    "q9RkXoty3X1fuaypDDRUi78rWgJWYJMmpJ",
+    "2Xvm6is5cAPA85xnSYXDuAqiRyoXiky5RaD",
+    "4CW2CPJEzxhn2PS4JoSLoWGL5QQ7dL2eji",
+    "24EG6uTzL7DHNzcwsygYGRR1nfu5kco7AZ1",
+    "KghGnWw5fppTrqHSERXZf61yf7GkuQdCnV",
+    "2WojewRA3LbpyXTP9ANy8CZqJMgmyNm3MDr",
+    "2BsMfywmGV3M2CoDA112Rs7ZBkiMHfy9X11",
+   	"kK1Q4gPyYfVVMzQtAPRzL8qXMqJ67Y7tKs",
+    "28J4mx8xfUtM92DbQ6i2Jmqw5J7dNivfroN",
+    "gQvgyG1djgtftoCVrSZmsRxr7okD4LheKw",
+    "3iFGBKapAWWzbiGFSr5ScbhrEPm6Esyvia",
+    "NFW2akQH2vu7AqkQXxFz2P5vkXTWkSqrSm",
+    "2MQJjLnWRp9eHh6MpCwpiUeshhtmri12mci",
+    "2QjRQUMyL6iodtHP9zKmxCNYZ7k3jxtk49C",
+    "USdfKy7B6oFNoauHWMmoCA7ND9rHqYw2Mf",
+   	"cA49et9WtptYHf6wA1F8qqVgH3kS5jJ9vK",
+   	"qaJT9TjcMi46sTKcgwRQU8o5Lw2Ea1gC4N",
+   	"22pyn5RyhqtTQu4obYjuWYRNNw4i54L8xVr",
+   	"22dkmukC6iH4FFLBmHne6modJZZQ3MC9BAT",
+   	"z6CJZfYLvmd41GRVE8HASjRcy5hqbpHZvE",
+   	"GEBWJ2KpRQDBTCCtvnaAJV2cYurgXS8pta",
+   	"oS8fbEm82cprmAeineBeDkaKd7QownDZQh",
+   	"rQpAs1LVQdphyj9ipEAuukAoj9kNpSP8cM",
+   	"6NSJKsPxmqipGAfFFhUKbkopjrvEESTX3j",
+   	"cuC68ycVXmD2EBzYFNYQ6akhKGrh3FGjSf",
+   	"bw4wtYU8toepomrhWP2p8UFYfHBbvEV425",
+   	"HvgNmDz5jD39Gwmi9VfDY1iYMhZUpZ8GKz",
+   	"SbApuZAYquWP3Q6iD51BcMBQjuApYEkRVf",
+   	"2Ugii5yxJgLzC59jV1vF8GK7UBZdvxwobeJ",
+   	"21N2iJ1qnQRiJWcEqNRxXwfNp8QcmiyhtPy",
+   	"9TC4RGs6AtFUsbcVWnSoCdoCpSfM66ALAc",
+   	"oQzn55UWG4iMcY9bTNb27aTnRdfiGHAwbD",
```

Comment:
dont know why these are showing up like this.
looks fine on my machine.

Reply:
github renders tabs wider